src/elements/_files/__name__.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule, Injector } from '@angular/core';
+import { createCustomElement } from '@angular/elements';
+import { <%= componentSymbolList %> } from '<%= barrel %>';
+
+@NgModule({
+  imports: [],
+  exports: [],
+  declarations: [],
+  entryComponents: [
+    <%= componentSymbolList %>
```

Comment:
Should this be a list? I'm not 100% sure about this but I was thinking that it was only possible to export a single component at a time as an Angular Element? I could be completely wrong about this and made the wrong assumption though, but figured I would mention it to be sure.

Reply:
List works well 🎉